### PR TITLE
FXIOS-1251 ⁃ [no issue] Add build channel attribute to Leanplum

### PR DESCRIPTION
**Pull request by dnarcese**
┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1251)

### CODE REVIEW

File: Client/Application/LeanplumIntegration.swift [hunk 1]
```diff
@@ -86,6 +86,7 @@ struct LPAttributeKey {
     static let experimentName = "Experiment name"
     static let experimentId = "Experiment id"
     static let experimentVariant = "Experiment variant"
+    static let buildChannelIsRelease = "Build channel is Release"
```
Comment (nbhasin2) [inline Client/Application/LeanplumIntegration.swift:89]:
```suggestion
    static let isReleaseBuild = "Build channel is Release"
```

File: Client/Application/LeanplumIntegration.swift [hunk 2]
```diff
@@ -212,7 +213,8 @@ class LeanPlumClient {
             LPAttributeKey.klarInstalled: klarInstalled(),
             LPAttributeKey.pocketInstalled: pocketInstalled(),
             LPAttributeKey.signedInSync: profile?.hasAccount() ?? false,
-            LPAttributeKey.fxaAccountVerified: profile?.hasSyncableAccount() ?? false
+            LPAttributeKey.fxaAccountVerified: profile?.hasSyncableAccount() ?? false,
+            LPAttributeKey.buildChannelIsRelease: AppConstants.BuildChannel == .release
```
Comment (nbhasin2) [inline Client/Application/LeanplumIntegration.swift:217]:
```suggestion
            LPAttributeKey.isReleaseBuild: AppConstants.BuildChannel == .release
```

**Review by nbhasin2 — APPROVED** [review]

**Comment by Otis1993** [comment]
hi 🚀

**Review by Otis1993 — COMMENTED** [review]
> @Otis1993

**Pull request merged**